addons/website/static/src/snippets/s_website_form/000.js

```diff
@@ -639,21 +635,13 @@ odoo.define('website.s_website_form', function (require) {
                 if (!dependencyIsVisible) {
                     return false;
                 }
-
                 const formData = new FormData(this.$target[0]);
-               
-                
-                const selected= formData.getAll(dependencyName);
-               
-                if(selected.includes(visibilityCondition))
-                {
-                 
-                return this._compareTo(comparator, visibilityCondition, visibilityCondition, between);
-                }
-                return this._compareTo(comparator, undefined, visibilityCondition, between);
-              
-                
-                
+                const currentValueOfDependency = formData.getAll(dependencyName);  
+                if(comparator=="contains" || comparator=="!contains")
```

Comment:
You can replace the double check by includes so you don't have to write twice comparator and should we add a third check it will be easier: ["contains", "!contains"].includes(comparator)

Reply:
I have made the changes